examples/stable/test/end_to_end/candid_rpc/workspaces/workspaces/w_descendant/canisters/different/package.json

```diff
@@ -0,0 +1,8 @@
+{
+    "dependencies": {
+        "azle": "file:../../../../test/test_packages/azle-999.999.999-w-descendant.tgz"
+    },
+    "scripts": {
+        "deploy": "cd dfx && dfx deploy w_descendant_different"
```

Comment:
add deep equivalents to the non-workspace examples

Reply:
done